PrmPkg/Samples/PrmSampleAcpiParameterBufferModule/PrmSampleAcpiParameterBufferModule.inf

```diff
@@ -39,7 +39,5 @@
 
 [BuildOptions.common]
   MSFT:*_*_*_DLINK_FLAGS  = /DLL /SUBSYSTEM:CONSOLE /VERSION:1.0
-  MSFT:*_*_*_GENFW_FLAGS = --keepoptionalheader
 
-  GCC:*_*_AARCH64_GENFW_FLAGS = --prm

```

Comment:
Why are we removing the PRM flag from PRM modules? All this does is make life harder if we at some point want to support PRM, because we have to go back through such commits to see which important flags were deleted. This also makes clear to me that GENFW_FLAGS should not be removed, as there is good reason for module definitions to modify it.

Reply:
We are removing GENFW_FLAGS because we have already removed GenFw from everywhere except BaseTools directory itself. If one day you decide to broaden ImageTool functionality by implementing some of the GenFw features, your life will be as hard as typing grep command in the console over the original edk2 repository (in order to find out which important flags you need).

Reply:
Yes, because I am aware of this particular change. There have been similar changes in the past I was not aware of and I wasted significant time trying to understand the status quo in AUDK, before deciding to check edk2 and finding the missing pieces.

Reply:
To be honest, PRMs are disgusting, and I would easily suggest removing them all along with the whole Pkg.

However, I agree with @mhaeuser that we should not make modifications to packages we neither test, nor build. PrmPkg is dead for us, and we do not test it. Please avoid making changes in it.

Reply:
@vit9696 Yes, removing the entire thing is fine to me, as then it's clear the tree has nothing interesting for it and I need to look in tianocore/edk2. But if the package remains and only the important build flags are gone, this is not clear at all (without prior knowledge of this commit), and I would waste time trying to understand the AUDK snapshot, before resorting to tianocore/edk2 only eventually. It's partial removals that are a pain.